pages/chaoss/visualizations/contrib_importance_pie.py

```diff
@@ -0,0 +1,256 @@
+from dash import html, dcc, callback
+import dash
+from dash import dcc
+import dash_bootstrap_components as dbc
+from dash.dependencies import Input, Output, State
+import plotly.graph_objects as go
+import pandas as pd
+import logging
+from dateutil.relativedelta import *  # type: ignore
+import plotly.express as px
+from pages.utils.graph_utils import get_graph_time_values, color_seq
+from queries.contributors_query import contributors_query as ctq
+import io
+from cache_manager.cache_manager import CacheManager as cm
+from pages.utils.job_utils import nodata_graph
+import time
+import random 
+
+PAGE = "chaoss"  
+VIZ_ID = "contrib-importance-pie"  
+
+gc_contrib_importance_pie = dbc.Card(
+    [
+        dbc.CardBody(
+            [
+                html.H3(
+                    "Contributor Importance by Action Type ",
+                    className="card-title",
+                    style={"textAlign": "center"},
+                ),
+                dbc.Popover(
+                    [
+                        dbc.PopoverHeader("Graph Info:"),
+                        dbc.PopoverBody("""
+                                        Visualizes the proportion of top k contributors and\n
+                                        remaining contributors according to a specific action\n
+                                        type. Contributors who are in the top k of contributions\n 
+                                        are anonynimzed by their unique alpha-numeric Contributor ID.\n
+                                        The remaining contributors are labeled as 'other'. Option to\n
+                                        filter out contributors that have 'bot' as their login username.
```

Comment:
Add "Note: Some commits have a contributor ID as none if there is no github account associated with the email used to commit"

Reply:
Added note - replaced "commit" to "contributions" to generalize